testproject/tests/test_second_step_authentication.py

```diff
@@ -809,39 +763,87 @@ def test_backup_codes_regeneration_disabled_method(
         )
     )
     response = client.post(
-        path="/auth/sms/codes/regenerate/",
+        path="/auth/sms_twilio/codes/regenerate/",
+        data={
+            "code": handler.create_code(),
+        },
         format="json",
     )
-    error_msg = "Method is disabled."
     assert response.status_code == 400
-    assert response.data.get("error") == error_msg
+    assert response.data.get("code")[0].code == "not_enabled"
 
 
 @pytest.mark.django_db
-def test_get_mfa_config():
-    client = APIClient()
-    response = client.get(
-        path="/auth/mfa/config/",
+def test_yubikey(active_user_with_yubi, offline_yubikey):
+    first_step_response = login(active_user_with_yubi)
+    handler = get_mfa_handler(mfa_method=active_user_with_yubi.mfa_methods.first())
+    response = APIClient().post(
+        path=PATH_AUTH_JWT_LOGIN_CODE,
+        data={
+            "ephemeral_token": first_step_response.data.get("ephemeral_token"),
+            "code": handler.create_code(),
+        },
         format="json",
     )
     assert response.status_code == 200
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too